eng/Version.Details.xml

```diff
@@ -29,94 +29,102 @@
     </Dependency>
     <Dependency Name="Microsoft.NET.Runtime.WorkloadTesting.Internal" Version="9.0.0-preview.3.24158.8">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>54d318d6da76e409d4e865220d9923283d55a06d</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.OpenApi" Version="9.0.0-preview.2.24128.4">
+    <Dependency Name="Microsoft.AspNetCore.App.Runtime.win-x64" Version="9.0.0-preview.5.24229.11">
       <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>e1ad9117a4dac3b0f5f2a7e2b10b43b7016379b9</Sha>
+      <Sha>207bc7d00c00bf454dc83789d6824609b1d8b02a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.OutputCaching.StackExchangeRedis" Version="9.0.0-preview.2.24128.4">
+    <Dependency Name="Microsoft.AspNetCore.OpenApi" Version="9.0.0-preview.5.24229.11">
       <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>e1ad9117a4dac3b0f5f2a7e2b10b43b7016379b9</Sha>
+      <Sha>207bc7d00c00bf454dc83789d6824609b1d8b02a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Caching.StackExchangeRedis" Version="9.0.0-preview.2.24128.4">
+    <Dependency Name="Microsoft.AspNetCore.OutputCaching.StackExchangeRedis" Version="9.0.0-preview.5.24229.11">
       <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>e1ad9117a4dac3b0f5f2a7e2b10b43b7016379b9</Sha>
+      <Sha>207bc7d00c00bf454dc83789d6824609b1d8b02a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="9.0.0-preview.2.24128.4">
+    <Dependency Name="Microsoft.Extensions.Caching.StackExchangeRedis" Version="9.0.0-preview.5.24229.11">
       <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>e1ad9117a4dac3b0f5f2a7e2b10b43b7016379b9</Sha>
+      <Sha>207bc7d00c00bf454dc83789d6824609b1d8b02a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Diagnostics.HealthChecks" Version="9.0.0-preview.2.24128.4">
+    <Dependency Name="Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="9.0.0-preview.5.24229.11">
       <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>e1ad9117a4dac3b0f5f2a7e2b10b43b7016379b9</Sha>
+      <Sha>207bc7d00c00bf454dc83789d6824609b1d8b02a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Features" Version="9.0.0-preview.2.24128.4">
+    <Dependency Name="Microsoft.Extensions.Diagnostics.HealthChecks" Version="9.0.0-preview.5.24229.11">
       <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>e1ad9117a4dac3b0f5f2a7e2b10b43b7016379b9</Sha>
+      <Sha>207bc7d00c00bf454dc83789d6824609b1d8b02a</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.Features" Version="9.0.0-preview.5.24229.11">
+      <Uri>https://github.com/dotnet/aspnetcore</Uri>
+      <Sha>207bc7d00c00bf454dc83789d6824609b1d8b02a</Sha>
     </Dependency>
     <Dependency Name="Microsoft.EntityFrameworkCore.Cosmos" Version="9.0.0-preview.2.24128.4">
```

Comment:
To make this coherent, we'll need to make an upstream change in dotnet/aspnetcore similar to https://github.com/dotnet/aspnetcore/pull/48398